example_configs/speech2text/w2l_large_8gpus.py

```diff
@@ -11,152 +11,165 @@
+            },
+            {
+                "type": "conv1d", "repeat": 3,
+                "kernel_size": [17], "stride": [1],
+                "num_channels": 512, "padding": "SAME",
+                "dropout_keep_prob": 0.8,
+            },
+            {
+                "type": "conv1d", "repeat": 3,
+                "kernel_size": [21], "stride": [1],
+                "num_channels": 640, "padding": "SAME",
+                "dropout_keep_prob": 0.7,
+            },
+            {
+                "type": "conv1d", "repeat": 3,
+                "kernel_size": [25], "stride": [1],
+                "num_channels": 768, "padding": "SAME",
+                "dropout_keep_prob": 0.7,
+            },
+            {
+                "type": "conv1d", "repeat": 1,
+                "kernel_size": [29], "stride": [1],
+                "num_channels": 896, "padding": "SAME",
+                "dropout_keep_prob": 0.6,
+            },
+            {
+                "type": "conv1d", "repeat": 1,
+                "kernel_size": [1], "stride": [1],
+                "num_channels": 1024, "padding": "SAME",
+                "dropout_keep_prob": 0.6,
+            },
+        ],
+
+        "dropout_keep_prob": 0.8,
+
+        "initializer": tf.contrib.layers.xavier_initializer,
+        "initializer_params": {
+            'uniform': False,
+        },
+        "normalization": "batch_norm",
+        "activation_fn": lambda x: tf.minimum(tf.nn.relu(x), 20.0),
+        "data_format": "channels_last",
+    },
+
+    "decoder": FullyConnectedCTCDecoder,
+    "decoder_params": {
+        "initializer": tf.contrib.layers.xavier_initializer,
+        "use_language_model": False,
+
+        # params for decoding the sequence with language model
+        "beam_width": 512,
+        "lm_weight": 2.0,
+        "word_count_weight": 1.5,
+        "valid_word_count_weight": 2.5,
+
+        "decoder_library_path": "ctc_decoder_with_lm/libctc_decoder_with_kenlm.so",
+        "lm_binary_path": "language_model/lm.binary",
+        "lm_trie_path": "language_model/trie",
+        "alphabet_config_path": "open_seq2seq/test_utils/toy_speech_data/vocab.txt",
+    },
+    "loss": CTCLoss,
+    "loss_params": {},
 }
 
 train_params = {
-  "data_layer": Speech2TextDataLayer,
-  "data_layer_params": {
-    "num_audio_features": 40,
-    "input_type": "logfbank",
-    "vocab_file": "open_seq2seq/test_utils/toy_speech_data/vocab.txt",
-    "dataset_files": [
-      "data/librispeech/librivox-train-clean-100.csv",
-      "data/librispeech/librivox-train-clean-360.csv",
-      "data/librispeech/librivox-train-other-500.csv",
-    ],
-    "shuffle": True,
-  },
+    "data_layer": Speech2TextDataLayer,
+    "data_layer_params": {
+        "num_audio_features": 64,
+        "input_type": "logfbank",
+        "vocab_file": "open_seq2seq/test_utils/toy_speech_data/vocab.txt",
+        "dataset_files": [
+            "/data/librispeech/librivox-train-clean-100.csv",
```

Comment:
Please set path prefix to "data" (instead of "/data")  to be consistent with other configs (DS2).

Reply:
Sure